src/storage/exec/IndexTopNNode.h

```diff
@@ -0,0 +1,94 @@
+/* Copyright (c) 2021 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License.
+ */
+#pragma once
+#include "folly/Likely.h"
+#include "storage/exec/IndexLimitNode.h"
+namespace nebula {
+namespace storage {
+template <class T>
+class TopNHeap final {
+ public:
+  ~TopNHeap() = default;
+
+  void setHeapSize(uint64_t heapSize) {
+    heapSize_ = heapSize;
+    v_.reserve(heapSize);
+  }
+
+  void setComparator(std::function<bool(T&, T&)> comparator) { comparator_ = comparator; }
+
+  void push(T data) {
```

Comment:
Maybe the performance will be better if you use rvalue

Reply:
yes, I have fixed it